client/src/components/GridField/GridFieldTable.js

```diff
@@ -55,7 +47,7 @@ class GridFieldTable extends Component {
 }
 
 GridFieldTable.propTypes = {
-  data: PropTypes.object,
+  data: PropTypes.object, // eslint-disable-line react/no-unused-prop-types
```

Comment:
```suggestion
```
It's better to just remove the unused prop rather than add a rule excepting it.

Reply:
Updated